### ISSUE TITLE
[Components and pattern standards] The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA. (04.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59489]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA.

### Issue Details
The URL for "Start a new message" page at present is: https://staging.va.gov/my-health/secure-messages/compose but the language "compose" is not used elsewhere in the product. Per our design system content guidance for URLs, "keywords should be selected from the primary keywords used in the H1 of the page and accurately represent the content of the page."

### VA.gov Experience Standard
[Category Number 04, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4 A

### Platform Recommendation
Update URL to /new-message (or something similar) to more closely match the language used in the page title.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by mnorthuis** [comment]
@oleksii-morgun-va My recommendation is to change this URL to www.va.gov/my-health/secure-messages/new-message/ to better align with the keywords utilized in the H1.

**Comment by bryan-riley-va** [comment]
JIRA ticket created: https://jira.devops.va.gov/browse/MHV-45982

**Comment by oleksii-morgun-va** [comment]
This issue has been resolved and deployed to staging. @mdewey assisted with validating

**Comment by oleksii-morgun-va** [comment]
This issue has been resolved and deployed to staging. @mdewey assisted with validating. Closing this ticket
